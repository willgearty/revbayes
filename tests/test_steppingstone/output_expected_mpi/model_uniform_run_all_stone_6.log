Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-23160.72	-23157.42	-3.300059	1.603545	0.409203	0.0897999	0.0334011	0.166093	0.18942	0.181353	0.153463	0.0401437	0.0919383	0.0515834	0.267591	0.175539	0.107706	0.0961259	0.222984	0.174736	0.156116	0.136936	0.154714	0.256419	0.10542	0.34207	0.232945	0.0245804	0.0992075	0.00671959	0.0131798	0.182839	0.0549477	0.0929653	0.0873097	0.0910496	0.115621	0.00501691	0.0276141	0.0477991	0.0227781	0.167503	0.167733	0.0156986	0.0119014	0.125898	0.187792	0.0778188	0.255354	0.0246707	0.0658983	0.52548	0.0507774	0.240884	0.604931	1.05596	2.09822	0.357378	0.361333	0.0826185	0.198671	0.4521661	5.383854
10	-23654.53	-23650.54	-3.989692	1.106507	0.302197	0.127553	0.361853	0.0848255	0.0840974	0.0803814	0.15093	0.0767504	0.0546139	0.0856524	0.105172	0.178415	0.147008	0.0560394	0.0732718	0.179024	0.186645	0.085663	0.251387	0.185214	0.0790728	0.0913789	0.126793	0.0350095	0.175786	0.00276284	0.0419256	0.040314	0.166429	0.294668	0.094831	0.224804	0.196196	0.00819099	0.132019	0.158625	0.0173266	0.036319	0.105918	0.0154618	0.00511258	0.198167	0.342412	0.0594439	0.198461	0.0497936	0.0819713	0.593264	0.0170666	0.157702	0.50658	1.01563	2.32009	0.33706	0.320106	0.0949183	0.247916	0.4018663	5.446217
