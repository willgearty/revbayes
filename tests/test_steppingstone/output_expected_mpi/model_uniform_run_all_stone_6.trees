Iteration	Posterior	Likelihood	Prior	psi
0	-23160.72	-23157.42	-3.300059	((((Perodicticus_potto[&index=18]:0.136936,Galago_senegalensis[&index=8]:0.040144)[&index=29]:0.054948,(Otolemur_crassicaudatus[&index=16]:0.174736,Nycticebus_coucang[&index=15]:0.222984)[&index=24]:0.024580)[&index=37]:0.022778,(((((Aotus_trivirgatus[&index=1]:0.409203,Cebus_albifrons[&index=3]:0.033401)[&index=35]:0.027614,(Saimiri_sciureus[&index=20]:0.256419,(Daubentonia_madagascariensis[&index=7]:0.153463,Varecia_variegata_variegata[&index=22]:0.342070)[&index=39]:0.167733)[&index=26]:0.006720)[&index=33]:0.115621,Callicebus_donacophilus[&index=2]:0.089800)[&index=43]:0.187792,Loris_tardigradus[&index=12]:0.175539)[&index=34]:0.005017,Lepilemur_hubbardorum[&index=11]:0.267591)[&index=28]:0.182839)[&index=41]:0.011901,Galeopterus_variegatus[&index=23]:0.232945,(Microcebus_murinus[&index=14]:0.096126,((Propithecus_coquereli[&index=19]:0.154714,Cheirogaleus_major[&index=4]:0.166093)[&index=27]:0.013180,(Lemur_catta[&index=10]:0.051583,(Tarsius_syrichta[&index=21]:0.105420,(Chlorocebus_aethiops[&index=5]:0.189420,(Colobus_guereza[&index=6]:0.181353,((Pan_paniscus[&index=17]:0.156116,Hylobates_lar[&index=9]:0.091938)[&index=42]:0.125898,Macaca_mulatta[&index=13]:0.107706)[&index=30]:0.092965)[&index=25]:0.099207)[&index=32]:0.091050)[&index=31]:0.087310)[&index=36]:0.047799)[&index=38]:0.167503)[&index=40]:0.015699)[&index=44];
10	-23654.53	-23650.54	-3.989692	((Nycticebus_coucang[&index=15]:0.073272,Otolemur_crassicaudatus[&index=16]:0.179024)[&index=24]:0.035010,((Galago_senegalensis[&index=8]:0.076750,Loris_tardigradus[&index=12]:0.178415)[&index=37]:0.017327,(Perodicticus_potto[&index=18]:0.085663,((((Cebus_albifrons[&index=3]:0.361853,Aotus_trivirgatus[&index=1]:0.302197)[&index=35]:0.132019,((Saimiri_sciureus[&index=20]:0.185214,(Daubentonia_madagascariensis[&index=7]:0.150930,Varecia_variegata_variegata[&index=22]:0.091379)[&index=39]:0.105918)[&index=26]:0.002763,Callicebus_donacophilus[&index=2]:0.127553)[&index=33]:0.196196)[&index=43]:0.342412,Lepilemur_hubbardorum[&index=11]:0.105172)[&index=34]:0.008191,(Microcebus_murinus[&index=14]:0.056039,((Tarsius_syrichta[&index=21]:0.079073,(((Pan_paniscus[&index=17]:0.186645,Chlorocebus_aethiops[&index=5]:0.084097)[&index=30]:0.294668,Macaca_mulatta[&index=13]:0.147008)[&index=25]:0.175786,(Hylobates_lar[&index=9]:0.054614,Colobus_guereza[&index=6]:0.080381)[&index=42]:0.198167)[&index=32]:0.224804)[&index=31]:0.094831,((Cheirogaleus_major[&index=4]:0.084825,Lemur_catta[&index=10]:0.085652)[&index=41]:0.005113,Propithecus_coquereli[&index=19]:0.251387)[&index=36]:0.158625)[&index=38]:0.036319)[&index=27]:0.041926)[&index=40]:0.015462)[&index=29]:0.166429)[&index=28]:0.040314,Galeopterus_variegatus[&index=23]:0.126793)[&index=44];
