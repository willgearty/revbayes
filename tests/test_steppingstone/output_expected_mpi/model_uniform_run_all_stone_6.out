state	power	likelihood
4	0.01024	-23248.1
5	0.01024	-23163.2
6	0.01024	-23199.9
7	0.01024	-23347.1
8	0.01024	-23314.5
9	0.01024	-23526.3
10	0.01024	-23650.5
